src/main/java/org/traccar/protocol/Vt200ProtocolDecoder.java

```diff
@@ -142,6 +142,32 @@ protected Object decode(
 
             return position;
 
+        } else if (type == 0x3089) {
```

Comment:
It looks like there's a lot of duplication with other GPS messages. Can we remove duplication?

Reply:
You mean replacing this line: https://github.com/traccar/traccar/blob/38f4c7ea61c58397ba42e527621bdece88fa472b/src/main/java/org/traccar/protocol/Vt200ProtocolDecoder.java#L70 

with this ` if (type == 0x2086 || type == 0x2084 || type == 0x2082 || type == 0x3089) {`? and use the common packages?

Reply:
Sure. Anything else?

Reply:
That's the only thing so far.

Reply:
Hi, the new version is available